### PR TITLE
Java: Add Webflux integration docs.

### LINKED ISSUE #3816: Document Spring Webflux integration support
### Core or SDK?

Platform/SDK

### Which part? Which one?

Sentry Java SDK
https://github.com/getsentry/sentry-java/pull/1529/

### Description

Install Spring Webflux integration

### Suggested Solution

Document Spring Webflux integration support

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
Fixes #3816

### CODE REVIEW

File: src/platforms/java/guides/spring-boot/webflux.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Webflux Integration
+sidebar_order: 6
+description: "Learn more about using Spring Webflux with Sentry."
+---
+
+Similar to Spring MVC, Sentry has a support for Spring Webflux. When developing a reactive web application, Sentry Spring Boot Starter creates infrastructure required for capturing errors triggered during HTTP request processing.
+
+<Note>
+Webflux support is available from version 5.1.0 in experimental phase. The API as well as internals may change in the future releases.
```
Comment (marandaneto) [inline src/platforms/java/guides/spring-boot/webflux.mdx:10]: that's fine, but we have a "predefined" wording for that (its part of our docs guidelines):

> Support for Flutter's Desktop is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.

we can either adapt the message above or keep it as it is, your call.

**Review by marandaneto — APPROVED** [review]

File: src/platforms/java/guides/spring-boot/webflux.mdx (hunk 1 above)
Comment (maciejwalkowiak) [reply src/platforms/java/guides/spring-boot/webflux.mdx:10]: I'll adapt

File: src/platforms/java/guides/spring-boot/record-user.mdx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 ---
 title: Record User Information
-sidebar_order: 6
+sidebar_order: 7
 description: "Learn about how you can record or customize the capture of user information."
 ---
 
+<Note>
+Recording user information is available only in Spring MVC integration.
```
Comment (imatwawana) [inline src/platforms/java/guides/spring-boot/record-user.mdx:8]:
```suggestion
Recording user information is available only in the Spring MVC integration.
```

File: src/platforms/java/guides/spring-boot/webflux.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Webflux Integration
```
Comment (imatwawana) [inline src/platforms/java/guides/spring-boot/webflux.mdx:2]: I've made edits to this new page, but I'm concerned that there's barely any content here. Will there be more added at some point? If not, can this page possibly be a small section on another page in the Spring Boot guide? 

```suggestion
title: WebFlux Integration
```

File: src/platforms/java/guides/spring-boot/webflux.mdx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+sidebar_order: 6
+description: "Learn more about using Spring Webflux with Sentry."
```
Comment (imatwawana) [inline src/platforms/java/guides/spring-boot/webflux.mdx:4]:
```suggestion
description: "Learn more about using Spring WebFlux with Sentry."
```

File: src/platforms/java/guides/spring-boot/webflux.mdx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+---
+
+Similar to Spring MVC, Sentry has a support for Spring Webflux. When developing a reactive web application, Sentry Spring Boot Starter creates infrastructure required for capturing errors triggered during HTTP request processing.
```
Comment (imatwawana) [inline src/platforms/java/guides/spring-boot/webflux.mdx:7]:
```suggestion
Similar to Spring MVC, Sentry offers support for Spring WebFlux. When developing a reactive web application, Sentry Spring Boot Starter creates the infrastructure required for capturing errors triggered during HTTP request processing.
```

File: src/platforms/java/guides/spring-boot/webflux.mdx [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+<Note>
+Support for Spring Webflux is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
```
Comment (imatwawana) [inline src/platforms/java/guides/spring-boot/webflux.mdx:10]:
```suggestion
Support for Spring WebFlux is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
```

**Review by imatwawana — COMMENTED** [review]
Minor edits, and I have a question about the placement of this content.

**Pull request merged**